client/packages/common/src/ui/layout/tables/columns/PaginationRow/PaginationRow.tsx

```diff
@@ -72,6 +88,20 @@ export const PaginationRow: FC<PaginationRowProps> = ({
             )}
           </Box>
 
+          <TablePagination
```

Comment:
Should probably just create a component, but this was quicker so I've put it in for now.
Will update, since all that's really needed is a select